app/views/decidim/action_delegator/_delegations_modal.html.erb

```diff
@@ -0,0 +1,93 @@
+<div class="reveal" data-reveal id="delegations-modal">
+  <div class="reveal__header">
+    <h3 class="reveal__title">
+      <%= t "action_delegator.delegations_modal.title", scope: "decidim" %>
+    </h3>
+    <button class="close-button" data-close aria-label="Close modal" type="button">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+
+  <p><%= t "action_delegator.delegations_modal.contextual_help", scope: "decidim" %></p>
+  <div class="card card--secondary">
+    <div class="card__content">
+
+      <% Decidim::ActionDelegator::ConsultationDelegations.for(question.consultation, current_user).each do |delegation| %>
+        <div class="row">
+          <div class="columns medium-5 medium-offset-1">
+            <%= delegation.granter.name %>
+          </div>
+          <div class="columns medium-5 medium-offset-1">
+
+            <%# Copied from decidim-consultation's _vote_button.html.erb %>
+            <% if question.consultation.upcoming? %>
+              <div class="button expanded disabled">
+                <div class="vote-button-caption"><%= t("questions.vote_button.vote", scope: "decidim") %></div>
+                <div>
+                  <%= t "questions.vote_button.starting_from",
+                        scope: "decidim",
+                        date: l(question.start_voting_date) %>
+                </div>
+              </div>
+            <% elsif question.consultation.finished? && signed_in? && question.voted_by?(delegation.granter) %>
+              <%= button_to decidim_consultations.question_question_votes_path(question),
+                          method: :delete,
+                          remote: true,
+                          data: { disable: true },
+                          class: "button expanded success disabled delegation_unvote_button" do %>
+                <div class="vote-button-caption">
+                  <%= t("questions.vote_button.already_voted", scope: "decidim") %>
+                </div>
+              <% end %>
+            <% elsif signed_in? && question.consultation.active? %>
+              <% if allowed_to? :unvote, :question, { question: question }, [Decidim::Consultations::Permissions, Decidim::Admin::Permissions, Decidim::Permissions], delegation.granter %>
```

Comment:
and here

Reply:
I'm not so sure about this one. IMO the mess with the permission chain should be solved by providing a new permission class of our own or similar so that we don't need to pass them here. As a result, this wouldn't diverge much from Decidim's calls to `allowed_to?`.

Reply:
true! in any case, it would be simplified!

Reply:
tracked in https://github.com/coopdevs/decidim-module-action_delegator/issues/24.